lnrpc/lightning.proto

```diff
@@ -2847,6 +2847,11 @@ message PendingChannelsResponse {
         // very likely canceled the funding and the channel will never become
         // fully operational.
         int32 funding_expiry_blocks = 3;
+
+        // The number of blocks remaining until the channel status changes from
+        // pending to active. A value of 0 indicates that the channel is now
+        // active.
+        int32 confirmations_until_active = 7;
```

Comment:
We should add two fields, the confirmation height and required_number_of_confirmations_until_acitve or something like this.